_books/tidy-first.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Tidy First?: A Personal Exercise in Empirical Software Design" 
+titleShort: "tidy-first"
+author: "Kent Beck"
+imageUrl: /assets/covers/tidyfirst.jpg
+tags:
+  - Software Quality
+audiences: 
+  - Developers
+pillars:
+  - "Technical Hygiene"
```

Comment:
```suggestion
  - "State of the art Software"
```

Reply:
This tag was meant for the Engineering Culture Pillars

Reply:
I could not find any doc about what tags / pillars to use, if you have some guidelines for it, maybe it can be added to the readme?